examples/simple-genai-server/README.md

```diff
@@ -0,0 +1,75 @@
+# Simple Game Server for GenAI
+
+A simple game server that serves as an example of how to integrate GenAI endpoints into your
+dedicated game server. You can either interact with one GenAI endpoint via TCP, or set up two
+endpoints to "chat" to each other.
+
+## Setting up the GenAI Inference Server
+
+You will need a separate GenAI inference server. This game server uses the
+[Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart) as its inference server.
+This particular inference server has the request structure for the /chat endpoint:
+
+```
+type GenAIRequest struct {
+	Context         string  `json:"context,omitempty"`
+	Prompt          string  `json:"prompt"`
+}
+```
+
+If you need a different request structure, you will need to clone or fork this repository and
+modify this in `main.go`. Update `REPOSITORY` in the `agones/examples/simple-genai-server/Makefile`
+to your own container registry and run `make build && make push` from within the
+`agones/examples/simple-genai-server` path. Then modify the `gameserver.yaml` to pull the image from
+your container registry. If you are making a series of changes you may also want to add
+`imagePullPolicy: Always` to the container image in `gameserver.yaml`.
+
+## Setting up the Game Server
+
+This example uses two separate clusters for the GenAI server and the game server. For the game
+server follow the instructions for [creating a cluster](https://agones.dev/site/docs/installation/creating-cluster/)
+and [installing Agones](https://agones.dev/site/docs/installation/install-agones/).
+
+Modify the `gameserver.yaml` `GenAiEndpoint` value to your inference server's endpoint. If you want
```

Comment:
Would it be better to have 2 yaml files? One for interactive mode and one for autonomous mode? That would simplify the instructions here a bit (pick the right file) and let this section focus on describing the configuration options.

Reply:
Good idea, I'll make separate example files.